examples/dcgan/README.md

```diff
@@ -0,0 +1,32 @@
+# DCGAN
+
+This is an example implementation of DCGAN (https://arxiv.org/abs/1511.06434) 
+trained on multi-GPU using `chainermn`.
+
+This code uses Cifar-10 dataset by default.
+You can use your own dataset by specifying `--dataset` argument to the directory consisting of image files for training.
+The model assumes the resolution of an input image is 32x32.
+If you want to use another image resolution, you need to change the network architecture in net.py.
+
+Below is an example learning result using cifar-10 dataset after 200 epoch, 
+where the model is trained using 4 GPUs with minibatch size 50 for each process.
+
+![example result](https://raw.githubusercontent.com/chainer/chainermn/master/examples/dcgan/example_image.png)
+
+## Implementation
+
+The original implementation is referenced from [chainer examples](https://github.com/chainer/chainer/tree/79d6bf6f4f5c86ba705b8fd377368519bc1fd264/examples/dcgan).
```

Comment:
I think that the URL of master is better than it.
The URL is: https://github.com/chainer/chainer/tree/master/examples/dcgan